docs/quickstart/actions-and-hooks.md

```diff
@@ -0,0 +1,193 @@
+---
+title: 6️⃣ Using Actions and Hooks in lakeFS
+description: lakeFS quickstart / Use Actions and Hooks to enforce conditions when committing and merging changes
+parent: ⭐ Quickstart ⭐
+nav_order: 30
+has_children: false
+next: ["Resources for learning more about lakeFS", "./learning-more-lakefs.html"]
+previous: ["Rollback the changes", "./rollback.html"]
+---
+
+# Actions and Hooks in lakeFS 🪝
+
+When we interact with lakeFS it can be useful to have certain checks performed at stages along the way. Let's see how [actions in lakeFS](/hooks/) can be of benefit here. 
+
+We're going to enforce a rule that when a commit is made to any branch that begins with `etl`: 
+
+* the commit message must not be blank
+* there must be `job_name` and `version` metadata
+* the `version` must be numeric
+
+To do this we'll create an _action_. In lakeFS, an action specifies one or more events that will trigger it, and references one or more _hooks_ to run when triggered. Actions are YAML files written to lakeFS under the `_lakefs_actions/` folder of the lakeFS repository.
+
+_Hooks_ can be either a [Lua](/hooks/lua.html) script that lakeFS will execute itself, an external [web hook](/hooks/webhook.html), or an [Airflow DAG](/hooks/airflow.html). In the example here we're using a Lua hook.
```

Comment:
Something about the current wording sounds off to my ear, but maybe it's just me :)
```diff
- In the example here we're using a Lua hook.
+ In this example, we'll use a Lua hook.
```

Reply:
thanks - fixed.